llumnix/global_scheduler/migration_scheduler.py

```diff
@@ -122,41 +173,25 @@ def _compute_instance_load_after_migrate(self, instance_info: InstanceInfo, is_m
 
 class DefragConstrained(PairMigrationPolicy):
     def pair_migration(self,
```

Comment:
It seems after this change, DefragConstrained's behavior has totally changed.
Do you mean a new PairMigrationPolicy?

Reply:
> It seems after this change, DefragConstrained's behavior has totally changed. Do you mean a new PairMigrationPolicy?

I did not change the behavior of DefragConstrained. I only unified its migration settings into the `_get_migration_settings` interface.

Reply:
is this true? @Xinyi-ECNU

Reply:
> It seems after this change, DefragConstrained's behavior has totally changed. Do you mean a new PairMigrationPolicy?



> is this true? @Xinyi-ECNU

DefragConstrained Policy has not been modified; it has only been refactored